tests/test_pytest_cov.py

```diff
@@ -1555,7 +1555,8 @@ def test_foo():
 
 SCRIPT_SIMPLE_RESULT = '4 * 100%'
 
-
+@pytest.mark.skipif('tuple(map(int, xdist.__version__.split("."))) >= (2, 5, 0)',
+                    reason="--boxed option was removed in version 2.5.0")
```

Comment:
It was deprecated in [2.5.0](https://github.com/pytest-dev/pytest-xdist/blob/master/CHANGELOG.rst#pytest-xdist-250-2021-12-10) and removed in [3.0.2](https://github.com/pytest-dev/pytest-xdist/blob/master/CHANGELOG.rst#pytest-xdist-302-2022-10-25).

```suggestion
@pytest.mark.skipif('tuple(map(int, xdist.__version__.split("."))) >= (3, 0, 2)',
                    reason="--boxed option was removed in version 3.0.2")
```